src/server/verification/verificationAPI.js

```diff
@@ -491,7 +491,7 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
       const { user, body } = req
       const verificationData: { code: string } = body.verificationData
       const { email } = user.otp || {}
-      const hashedNewEmail = sha3(email)
+      const hashedNewEmail = email ? sha3(email) : null
```

Comment:
what is the reason that we get empty email from client side? has it been addressed?

Reply:
from the client-side we get the otp code
the email we get from the user object

Reply:
ok sorry, still the question is why is the data is malformed in the first place? why is the user trying to verify an email that we dont have?